python/tvm/contrib/graph_executor.py

```diff
@@ -195,6 +198,49 @@ def set_input(self, key=None, value=None, **params):
                 if val:
                     self._get_input(k).copyfrom(params[k])
 
+    def set_output(self, key, value):
+        """Set outputs to the module
+
+        Parameters
+        ----------
+        key : int or str
+           The output key
+
+        value : the output value
+           The output value
+        """
+        self._set_output(key, value)
```

Comment:
this is not needed, it doesn't make sense to copy external data to the output tensor